ui/src/services/segmenter/SegmenterScope.js

```diff
@@ -0,0 +1,13 @@
+export const getSegmenterScope = (scope) => {
```

Comment:
Added this file to match `ui/src/services/segmenter/SegmenterStatus.js`. The definitions will be taken from here instead of inline.